src/web/app/components/teammates-common/submission-status-tooltip.pipe.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { FeedbackSessionSubmissionStatus } from '../../../types/api-output';
+import { SubmissionStatusTooltipPipe } from './submission-status-tooltip.pipe';
+
+describe('SubmissionStatusTooltipPipe', () => {
+  it('create an instance', () => {
+    const pipe: SubmissionStatusTooltipPipe = new SubmissionStatusTooltipPipe();
+    expect(pipe).toBeTruthy();
+  });
+
+  it('transform with the tooltip description correctly', () => {
+    const pipe: SubmissionStatusTooltipPipe = new SubmissionStatusTooltipPipe();
+    expect(pipe).toBeTruthy();
+  });
```

Comment:
Did you forget to write this test case? It does not do what it says. Or is this superceded by `transform with deadlines correctly`? Maybe you forgot to remove?

Reply:
Yup, sorry this was unintentional